pkg/eventcounter/api/api_test.go

```diff
@@ -1083,17 +1085,26 @@ func TestGetEvaluationTimeseriesCountV2(t *testing.T) {
 					&featureproto.GetFeatureResponse{
 						Feature: &featureproto.Feature{
 							Id:         "fid",
-							Variations: []*featureproto.Variation{{Id: "vid0"}, {Id: "vid1"}},
+							Variations: []*featureproto.Variation{{Id: vID0}, {Id: vID1}},
 						},
 					}, nil)
-				s.evaluationCountCacher.(*eccachemock.MockEventCounterCache).EXPECT().GetEventCounts(gomock.Any()).Return(
-					[]float64{
-						1, 3, 5,
-					}, nil).AnyTimes()
-				s.evaluationCountCacher.(*eccachemock.MockEventCounterCache).EXPECT().GetUserCounts(gomock.Any()).Return(
-					[]float64{
-						2, 4, 6,
-					}, nil).AnyTimes()
+				vIDs := []string{vID0, vID1, defaultVariationID}
+				endAt := time.Now()
+				startAt, err := genInterval(jpLocation, endAt, 30)
+				assert.NoError(t, err)
+				timeStamps := getOneMonthTimeStamps(startAt)
+				for _, vID := range vIDs {
+					ec := getEventCountKeys(vID, fID, environmentNamespace, timeStamps)
+					s.evaluationCountCacher.(*eccachemock.MockEventCounterCache).EXPECT().GetEventCounts(ec).Return(
+						[]float64{
+							1, 3, 5,
+						}, nil).AnyTimes()
```

Comment:
Nice!
Can you remove the `AnyTimes` for both mocks? To ensure that the code is not calling more than it should.